src/makeRobot.js

```diff
@@ -38,7 +38,60 @@
  * @return {Robot}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  const robot = {
+    name,
+    wheels,
+    version,
+    coords: {
+      x: 0,
+      y: 0,
+    },
+    get info() {
+      // eslint-disable-next-line max-len
+      return `name: ${this.name}, chip version: ${this.version}, wheels: ${this.wheels}`;
+    },
+    get location() {
+      return `${this.name}: x=${this.coords.x}, y=${this.coords.y}`;
+    },
+    goRight(steps = 1) {
```

Comment:
Add spaces between methods to improve the readability 
```suggestion
    },
    
    get location() {
      return `${this.name}: x=${this.coords.x}, y=${this.coords.y}`;
    },
    
    goRight(steps = 1) {
```